packer/scripts/vagrant.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+# install the insecure public key for Vagrant
+mkdir -p /home/vagrant/.ssh
+curl https://raw.githubusercontent.com/mitchellh/vagrant/master/keys/vagrant.pub > /home/vagrant/.ssh/authorized_keys
```

Comment:
I am seeing an error when running `vagrant rsync`:

```
    default: Inserting generated public key within guest...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

chmod 0700 ~/.ssh

Stdout from the command:



Stderr from the command:
```

Running the following on the vagrant box fixed it for me:

```shell
sudo chmod 0700 ~/.ssh
sudo chown vagrant:vagrant .ssh
sudo chmod 600 ~/.ssh/authorized_keys
sudo chown vagrant:vagrant ~/.ssh/authorized_keys
```

Think we need to add those to the packer build (see http://docs.vagrantup.com/v2/boxes/base.html)